pkgs/os-specific/linux/msi-ec/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  linuxPackages,
+  pkgs,
```

Comment:
I could be wrong here, but can we not simply import `git` instead of `pkgs` and replace `pkgs.git` -> `git` later in the derivation?